client/types.go

```diff
@@ -0,0 +1,52 @@
+package client
+
+import (
+	"context"
+
+	"github.com/tigrisdata/tigrisdb-client-go/driver"
+)
+
+// TransactionFunc is a user-provided function that will be run
+// within the context of a tranaction.
+type TransactionFunc func(
+	ctx context.Context,
+	tr driver.Tx,
+) (interface{}, error)
+
+// Client is the interface for a TigrisDB client.
+type Client interface {
```

Comment:
move this interface to the client.go?

Reply:
shouldnt interfaces be declared in types.go? Thats what its for I thought (Thats how ive used it in every codebase that has types.go)